server/models/services/AtService.js

```diff
@@ -194,14 +194,14 @@ const removeAt = async (id, deleteOptions = { truncate: false }) => {
  * @returns {Promise<*>}
  */
 const getAtVersionByQuery = async (
-    { atId, atVersion },
+    { atId, atVersion, availability = null },
     atVersionAttributes = AT_VERSION_ATTRIBUTES,
     atAttributes = AT_ATTRIBUTES,
     options = {}
 ) => {
     return ModelService.getByQuery(
         AtVersion,
-        { atId, atVersion },
+        { atId, atVersion, ...(availability && { availability }) },
```

Comment:
Not sure if this property is ever likely to be used when querying but shouldn't affect anything. You should also be able to pass it as a query even if `null` so may not need the null check here.